expression/explain.go

```diff
@@ -170,6 +242,46 @@ func SortedExplainNormalizedExpressionList(exprs []Expression) []byte {
 	return sortedExplainExpressionList(exprs, true)
 }
 
+// SortedExplainExpressionList generates explain information for a list of expressions in order.
+// In some scenarios, the expr's order may not be stable when executing multiple times.
+// So we add a sort to make its explain result stable.
+func SortedExplainExpressionList4InList(exprs []Expression) []byte {
+	return sortedExplainExpressionList4InList(exprs, false)
+}
+
+func sortedExplainExpressionList4InList(exprs []Expression, normalized bool) []byte {
+	buffer := bytes.NewBufferString("")
+	exprInfos := make([]string, 0, len(exprs))
+	for _, expr := range exprs {
+		if _, ok := expr.(*ScalarFunction); ok {
+			if normalized {
+				exprInfos = append(exprInfos, expr.ExplainNormalizedInfo4InList())
+			} else {
+				exprInfos = append(exprInfos, expr.ExplainInfo4InList())
+			}
```

Comment:
Is it possible to embedded this part to `sortedExplainExpressionList` to avoid some duplicated code?

Reply:
Done